features_and_tools/firely-server-ingest.rst

```diff
@@ -3,6 +3,10 @@
 Bulk Import via Firely Server Ingest
 ====================================
 
+.. note::
```

Comment:
Should we mention the separate license token /unlimited?

Reply:
No, it already mentions the limit of the eval edition in the text.